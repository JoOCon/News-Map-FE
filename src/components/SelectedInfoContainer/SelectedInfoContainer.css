.selected-wrapper {
  display: flex;
  transition: 2s; }

.selected-container {
  padding: 0 1.5em;
  width: 100%;
  font-family: caviar;
  letter-spacing: 2px;
  font-size: 1.1em;
  display: flex;
  flex-direction: column; }

.weather-high {
  color: #ec7373;
  margin: auto 0.2rem; }

.weather-low {
  color: #b9b9b9;
  margin: auto 0.2rem; }

.weather-precip {
  margin: 1rem;
  color: #76a595; }

.weather-info {
  display: flex;
  justify-content: center; }

.weather-container {
  display: flex;
  border: 0 2px 2px 2px solid black;
  border: 2px solid #7c7c7c;
  border-top: 0;
  border-radius: 2rem;
  border-radius: 0rem 0rem 2rem 2rem;
  padding: 0.5rem; }

.selected-date {
  font-size: 4rem;
  color: #acabab; }

.selected-venue {
  font-size: 2.5rem;
  color: #020202; }

.selected-distance {
  font-size: 1.3rem;
  color: #222222; }

.selected-address {
  font-size: 2rem;
  color: #c7c7c7; }

.selected-remove {
  font-family: caviar;
  font-weight: 800;
  background: transparent;
  padding: 0.5rem 1rem;
  font-size: 1.1rem;
  letter-spacing: 1.3px;
  border-radius: 1rem;
  border-color: #383535;
  cursor: pointer;
  margin: 2rem; }

.selected-remove:hover {
  background-color: #686868;
  color: #a39191; }

.selected-link {
  font-weight: 800;
  background: transparent;
  padding: 0.5rem 1rem;
  font-size: 1.1rem;
  letter-spacing: 1.3px;
  border: 1px solid black;
  border-radius: 1rem;
  border-color: #383535;
  cursor: pointer;
  text-decoration: none;
  color: black; }

.selected-link:hover {
  background-color: #686868;
  color: #90aab9; }
