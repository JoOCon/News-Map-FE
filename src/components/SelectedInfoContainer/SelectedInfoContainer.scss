.selected-wrapper {
  display: flex;
  transition: 2s;
}

.selected-container {
  padding: 2em 1em;
  width: 20em;
  font-family: caviar;
  letter-spacing: 2px;
  font-size: 1.1em;
}

.weather-high {
  color: rgb(194, 129, 129);
  margin: auto 0.2rem;
}

.weather-low {
  color: rgb(74, 74, 131);
  margin: auto 0.2rem;
}

.weather-info {
  display: flex;
}
