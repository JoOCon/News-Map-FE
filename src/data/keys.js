export const googleClientId =
  '241779944002-qa42v6cvk3kpu8da0jimdr364vn41t1r.apps.googleusercontent.com';
