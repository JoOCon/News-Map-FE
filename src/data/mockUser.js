export const mockUser = {
  El: '117977763637801404146',
  Zi: {
    token_type: 'Bearer',
    access_token:
      'ya29.GlxBBtt0Yi5X7awZrvhwo7UAQjrOLQ_JwGQ0ZVcJhahzVb4kj_Z5NqMx4wAhouEXxtNs72jp9pG6TZ68H1nf3bgknrUoiWzhRnotssJFcYMiopKmF1oc0ejm8ZDCwQ',
    scope:
      'openid email profile https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/plus.me https://www.googleapis.com/auth/userinfo.profile',
    login_hint:
      'AJDLj6JUa8yxXrhHdWRHIV0S13cArCX2skrQ7skgtwzDVxi66KXli5pV9QkfHh_W71S4j789mYsJSJTDHdBzZgGY2XexHHgGyA',
    expires_in: 3600,
    id_token:
      'eyJhbGciOiJSUzI1NiIsImtpZCI6IjcyOGY0MDE2NjUyMDc5YjllZDk5ODYxYmIwOWJhZmM1YTQ1YmFhODYiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJhY2NvdW50cy5nb29nbGUuY29tIiwiYXpwIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwiYXVkIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTE3OTc3NzYzNjM3ODAxNDA0MTQ2IiwiZW1haWwiOiJweWtpbTA1OTFAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImF0X2hhc2giOiJNVS16RFc2c3lqbEhTNDZsbW5vMUVnIiwibmFtZSI6IlBhdWwgSyIsInBpY3R1cmUiOiJodHRwczovL2xoNC5nb29nbGV1c2VyY29udGVudC5jb20vLXE3OWJ2NE5jX2RZL0FBQUFBQUFBQUFJL0FBQUFBQUFBQUFBL0FCdE5sYkRPMEIyU3A5U3pPWGVxZ0sxOWJvcXNUaUlWQUEvczk2LWMvcGhvdG8uanBnIiwiZ2l2ZW5fbmFtZSI6IlBhdWwiLCJmYW1pbHlfbmFtZSI6IksiLCJsb2NhbGUiOiJlbiIsImlhdCI6MTU0MDUwODUwMCwiZXhwIjoxNTQwNTEyMTAwLCJqdGkiOiI2ZTYyNTg0Mzc2NTYwNDMwNjA1Y2MxYmVlNTUxOTE5ZmNjNTVmN2VmIn0.HFfyTNUGKeLor17A-qWqSRNxfITUysVYoPwC2UFzbn06Ulx4GEj7NDm3qMXFQCxXO50dGvOcsF8MKvB2fsmRkdVZeoin5QCdZa43xbfxoTmUCPX-zKFNUt8Ln3rcewNuQubkZ15Tq1BvaIh-vOQLxGSi7_yZXhTnnGLz1QNYaIZkdMUILJkGlOqigCN68GgumUWePIWn73Z-OGeKV0AStyLWpXRYtO6gIv1o8H_XV1WT2Iud-sTINDKWSK9iuomoFEnAHRcxbOQDX9A9yHHv7ZxIyzzKTd1QQmxnB_Aiux0mxu_zsZiZU5J9LS24QV5_EaAgWVKj3nCLYytEOGbjJQ',
    session_state: { extraQueryParams: { authuser: '0' } },
    first_issued_at: 1540508499764,
    expires_at: 1540512099764,
    idpId: 'google'
  },
  w3: {
    Eea: '117977763637801404146',
    ig: 'Paul K',
    ofa: 'Paul',
    wea: 'K',
    Paa:
      'https://lh4.googleusercontent.com/-q79bv4Nc_dY/AAAAAAAAAAI/AAAAAAAAAAA/ABtNlbDO0B2Sp9SzOXeqgK19boqsTiIVAA/s96-c/photo.jpg',
    U3: 'pykim0591@gmail.com'
  },
  googleId: '117977763637801404146',
  tokenObj: {
    token_type: 'Bearer',
    access_token:
      'ya29.GlxBBtt0Yi5X7awZrvhwo7UAQjrOLQ_JwGQ0ZVcJhahzVb4kj_Z5NqMx4wAhouEXxtNs72jp9pG6TZ68H1nf3bgknrUoiWzhRnotssJFcYMiopKmF1oc0ejm8ZDCwQ',
    scope:
      'openid email profile https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/plus.me https://www.googleapis.com/auth/userinfo.profile',
    login_hint:
      'AJDLj6JUa8yxXrhHdWRHIV0S13cArCX2skrQ7skgtwzDVxi66KXli5pV9QkfHh_W71S4j789mYsJSJTDHdBzZgGY2XexHHgGyA',
    expires_in: 3600,
    id_token:
      'eyJhbGciOiJSUzI1NiIsImtpZCI6IjcyOGY0MDE2NjUyMDc5YjllZDk5ODYxYmIwOWJhZmM1YTQ1YmFhODYiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJhY2NvdW50cy5nb29nbGUuY29tIiwiYXpwIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwiYXVkIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTE3OTc3NzYzNjM3ODAxNDA0MTQ2IiwiZW1haWwiOiJweWtpbTA1OTFAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImF0X2hhc2giOiJNVS16RFc2c3lqbEhTNDZsbW5vMUVnIiwibmFtZSI6IlBhdWwgSyIsInBpY3R1cmUiOiJodHRwczovL2xoNC5nb29nbGV1c2VyY29udGVudC5jb20vLXE3OWJ2NE5jX2RZL0FBQUFBQUFBQUFJL0FBQUFBQUFBQUFBL0FCdE5sYkRPMEIyU3A5U3pPWGVxZ0sxOWJvcXNUaUlWQUEvczk2LWMvcGhvdG8uanBnIiwiZ2l2ZW5fbmFtZSI6IlBhdWwiLCJmYW1pbHlfbmFtZSI6IksiLCJsb2NhbGUiOiJlbiIsImlhdCI6MTU0MDUwODUwMCwiZXhwIjoxNTQwNTEyMTAwLCJqdGkiOiI2ZTYyNTg0Mzc2NTYwNDMwNjA1Y2MxYmVlNTUxOTE5ZmNjNTVmN2VmIn0.HFfyTNUGKeLor17A-qWqSRNxfITUysVYoPwC2UFzbn06Ulx4GEj7NDm3qMXFQCxXO50dGvOcsF8MKvB2fsmRkdVZeoin5QCdZa43xbfxoTmUCPX-zKFNUt8Ln3rcewNuQubkZ15Tq1BvaIh-vOQLxGSi7_yZXhTnnGLz1QNYaIZkdMUILJkGlOqigCN68GgumUWePIWn73Z-OGeKV0AStyLWpXRYtO6gIv1o8H_XV1WT2Iud-sTINDKWSK9iuomoFEnAHRcxbOQDX9A9yHHv7ZxIyzzKTd1QQmxnB_Aiux0mxu_zsZiZU5J9LS24QV5_EaAgWVKj3nCLYytEOGbjJQ',
    session_state: { extraQueryParams: { authuser: '0' } },
    first_issued_at: 1540508499764,
    expires_at: 1540512099764,
    idpId: 'google'
  },
  tokenId:
    'eyJhbGciOiJSUzI1NiIsImtpZCI6IjcyOGY0MDE2NjUyMDc5YjllZDk5ODYxYmIwOWJhZmM1YTQ1YmFhODYiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJhY2NvdW50cy5nb29nbGUuY29tIiwiYXpwIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwiYXVkIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTE3OTc3NzYzNjM3ODAxNDA0MTQ2IiwiZW1haWwiOiJweWtpbTA1OTFAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImF0X2hhc2giOiJNVS16RFc2c3lqbEhTNDZsbW5vMUVnIiwibmFtZSI6IlBhdWwgSyIsInBpY3R1cmUiOiJodHRwczovL2xoNC5nb29nbGV1c2VyY29udGVudC5jb20vLXE3OWJ2NE5jX2RZL0FBQUFBQUFBQUFJL0FBQUFBQUFBQUFBL0FCdE5sYkRPMEIyU3A5U3pPWGVxZ0sxOWJvcXNUaUlWQUEvczk2LWMvcGhvdG8uanBnIiwiZ2l2ZW5fbmFtZSI6IlBhdWwiLCJmYW1pbHlfbmFtZSI6IksiLCJsb2NhbGUiOiJlbiIsImlhdCI6MTU0MDUwODUwMCwiZXhwIjoxNTQwNTEyMTAwLCJqdGkiOiI2ZTYyNTg0Mzc2NTYwNDMwNjA1Y2MxYmVlNTUxOTE5ZmNjNTVmN2VmIn0.HFfyTNUGKeLor17A-qWqSRNxfITUysVYoPwC2UFzbn06Ulx4GEj7NDm3qMXFQCxXO50dGvOcsF8MKvB2fsmRkdVZeoin5QCdZa43xbfxoTmUCPX-zKFNUt8Ln3rcewNuQubkZ15Tq1BvaIh-vOQLxGSi7_yZXhTnnGLz1QNYaIZkdMUILJkGlOqigCN68GgumUWePIWn73Z-OGeKV0AStyLWpXRYtO6gIv1o8H_XV1WT2Iud-sTINDKWSK9iuomoFEnAHRcxbOQDX9A9yHHv7ZxIyzzKTd1QQmxnB_Aiux0mxu_zsZiZU5J9LS24QV5_EaAgWVKj3nCLYytEOGbjJQ',
  accessToken:
    'ya29.GlxBBtt0Yi5X7awZrvhwo7UAQjrOLQ_JwGQ0ZVcJhahzVb4kj_Z5NqMx4wAhouEXxtNs72jp9pG6TZ68H1nf3bgknrUoiWzhRnotssJFcYMiopKmF1oc0ejm8ZDCwQ',
  profileObj: {
    googleId: '117977763637801404146',
    imageUrl:
      'https://lh4.googleusercontent.com/-q79bv4Nc_dY/AAAAAAAAAAI/AAAAAAAAAAA/ABtNlbDO0B2Sp9SzOXeqgK19boqsTiIVAA/s96-c/photo.jpg',
    email: 'pykim0591@gmail.com',
    name: 'Paul K',
    givenName: 'Paul',
    familyName: 'K'
  }
};

export const mockStateUser = {
  first_name: 'Paul',
  last_name: 'K',
  email: 'pykim0591@gmail.com',
  gid: '117977763637801404146',
  id: 3,
  created_at: '2018-10-24T23:37:44.724Z',
  updated_at: '2018-10-24T23:37:44.724Z'
};

export const mockUserResponse = {
  created_at: '2018-10-24T23:37:44.724Z',
  email: 'pykim0591@gmail.com',
  first_name: 'Paul',
  gid: '117977763637801404146',
  id: 3,
  last_name: 'K',
  updated_at: '2018-10-24T23:37:44.724Z'
};
