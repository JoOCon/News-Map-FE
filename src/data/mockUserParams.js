export const responseGoogle1 =
  'https://event-mapper-api.herokuapp.com/api/v1/users';

export const responseGoogle2 = {
  body:
    '{"user":{"google_id":"117977763637801404146","given_name":"Paul","family_name":"K","email":"pykim0591@gmail.com"}}',
  headers: { 'Content-Type': 'application/json' },
  method: 'POST'
};

export const getEvents =
  'https://app.ticketmaster.com/discovery/v2/events.json?apikey=GddBLHQGZLJjPNWsNxOrYRAoBx8Hdqrf&geoPoint=9wjr4et3f&radius=200&size=100';

export const postUser = {
  body: '{"user":{"email":"pykim0591@gmail.com"}}',
  headers: { 'Content-Type': 'application/json' },
  method: 'POST'
};

export const setFavorite = {
  body: '{}',
  headers: { 'Content-Type': 'application/json' },
  method: 'POST'
};

export const removeEvent =
  'https://event-mapper-api.herokuapp.com/api/v1/users/undefined/events/undefined';

export const removeEventOptions = {
  headers: { 'Content-Type': 'application/json' },
  method: 'DELETE'
};
