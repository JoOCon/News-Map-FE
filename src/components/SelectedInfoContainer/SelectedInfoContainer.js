import React from 'react';
import PropTypes from 'prop-types';
import './SelectedInfoContainer.css';

export const SelectedInfoContainer = ({ removeEvent, item, weather }) => {
  return (
    <div className="selected-container">
      <img src={require(`../../images/${weather.icon}.png`)} />
      <p className="selected selected-date">{item.date}</p>
      <p className="selected selected-distance">{item.distance}m</p>
      <p className="selected selected-date">{item.address}</p>
      <p className="selected selected-date">{item.venue_name}</p>
      <a href={item.url}>Event Info</a>
      <button
        className="selected selected-date"
        onClick={removeEvent.bind(null, item)}
      >
        Remove from Watchlist
      </button>
    </div>
  );
};

SelectedInfoContainer.propTypes = {
  displayInfo: PropTypes.object
};
