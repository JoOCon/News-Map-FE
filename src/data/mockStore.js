import * as user from './mockUser';
import * as events from './mockEvents';

export const mockStore = {
  activeUser: user.mockStateUser,
  userLocation: { latitude: 39.43243, longitude: -104.98434 },
  watchList: events.userWatchlist,
  events: events.eventsResponse
};
