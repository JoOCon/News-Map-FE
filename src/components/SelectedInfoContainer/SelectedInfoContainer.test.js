import React from 'react';

import { SelectedInfoContainer } from './SelectedInfoContainer';
import * as events from '../../data/mockEvents';

describe('SelectedInfoContainer', () => {
  let wrapper;
  let mockRemove;

  beforeEach(() => {
    mockRemove = jest.fn();
    wrapper = shallow(
      <SelectedInfoContainer
        removeEvent={mockRemove}
        item={events.mockTargetEventFalse}
        weather={{ icon: 'cloudy' }}
      />
    );
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should call removeEvent onClick', () => {
    wrapper.find('button').simulate('click');

    expect(mockRemove).toHaveBeenCalled();
  });
});
