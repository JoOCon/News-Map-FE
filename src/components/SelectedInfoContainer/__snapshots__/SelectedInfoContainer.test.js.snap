// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedInfoContainer should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SelectedInfoContainer
    item={
      Object {
        "address": "1510 Clarkson Denver, CO 80218",
        "date": "Wed, Oct 31, 2018 6:30 PM",
        "distance": 0.5,
        "e_id": "G5vzZ4C58cdb-",
        "favorite": false,
        "img": "https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg",
        "lat": "39.740611",
        "lng": "-104.977194",
        "name": "Danzig - 30 Year Anniversary Tour",
        "url": "http://concerts.livenation.com/event/1E0054E2D0B43621",
        "venue_name": "Fillmore Auditorium (Denver)",
      }
    }
    removeEvent={[MockFunction]}
    weather={
      Object {
        "icon": "cloudy",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          src="cloudy.png"
        />,
        <p
          className="selected selected-date"
        >
          Wed, Oct 31, 2018 6:30 PM
        </p>,
        <p
          className="selected selected-distance"
        >
          0.5
          m
        </p>,
        <p
          className="selected selected-date"
        >
          1510 Clarkson Denver, CO 80218
        </p>,
        <p
          className="selected selected-date"
        >
          Fillmore Auditorium (Denver)
        </p>,
        <a
          href="http://concerts.livenation.com/event/1E0054E2D0B43621"
        >
          Event Info
        </a>,
        <button
          className="selected selected-date"
          onClick={[Function]}
        >
          Remove from Watchlist
        </button>,
      ],
      "className": "selected-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "src": "cloudy.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Wed, Oct 31, 2018 6:30 PM",
          "className": "selected selected-date",
        },
        "ref": null,
        "rendered": "Wed, Oct 31, 2018 6:30 PM",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            0.5,
            "m",
          ],
          "className": "selected selected-distance",
        },
        "ref": null,
        "rendered": Array [
          0.5,
          "m",
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "1510 Clarkson Denver, CO 80218",
          "className": "selected selected-date",
        },
        "ref": null,
        "rendered": "1510 Clarkson Denver, CO 80218",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Fillmore Auditorium (Denver)",
          "className": "selected selected-date",
        },
        "ref": null,
        "rendered": "Fillmore Auditorium (Denver)",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Event Info",
          "href": "http://concerts.livenation.com/event/1E0054E2D0B43621",
        },
        "ref": null,
        "rendered": "Event Info",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Remove from Watchlist",
          "className": "selected selected-date",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": "Remove from Watchlist",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            src="cloudy.png"
          />,
          <p
            className="selected selected-date"
          >
            Wed, Oct 31, 2018 6:30 PM
          </p>,
          <p
            className="selected selected-distance"
          >
            0.5
            m
          </p>,
          <p
            className="selected selected-date"
          >
            1510 Clarkson Denver, CO 80218
          </p>,
          <p
            className="selected selected-date"
          >
            Fillmore Auditorium (Denver)
          </p>,
          <a
            href="http://concerts.livenation.com/event/1E0054E2D0B43621"
          >
            Event Info
          </a>,
          <button
            className="selected selected-date"
            onClick={[Function]}
          >
            Remove from Watchlist
          </button>,
        ],
        "className": "selected-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "src": "cloudy.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Wed, Oct 31, 2018 6:30 PM",
            "className": "selected selected-date",
          },
          "ref": null,
          "rendered": "Wed, Oct 31, 2018 6:30 PM",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              0.5,
              "m",
            ],
            "className": "selected selected-distance",
          },
          "ref": null,
          "rendered": Array [
            0.5,
            "m",
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "1510 Clarkson Denver, CO 80218",
            "className": "selected selected-date",
          },
          "ref": null,
          "rendered": "1510 Clarkson Denver, CO 80218",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Fillmore Auditorium (Denver)",
            "className": "selected selected-date",
          },
          "ref": null,
          "rendered": "Fillmore Auditorium (Denver)",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Event Info",
            "href": "http://concerts.livenation.com/event/1E0054E2D0B43621",
          },
          "ref": null,
          "rendered": "Event Info",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Remove from Watchlist",
            "className": "selected selected-date",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "Remove from Watchlist",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
